index.html

```diff
@@ -17,7 +17,7 @@
 <body>
     <div class="game-wrapper">
         <header class="header">
-            <div class="header__game-logo"></div>
+            <img class="header__game-logo" src="static/assets/img/logo.png"></img>
```

Comment:
This element will probably be created dynamically in js by element creator. In this case, you would be needing a function that creates this element, adds listener to it, and returns it.